client/state/analytics/actions.js

```diff
@@ -24,7 +24,7 @@ const joinAnalytics = ( analytics, action ) =>
 export const composeAnalytics = ( ...analytics ) => ( {
 	type: ANALYTICS_MULTI_TRACK,
 	meta: {
-		analytics: analytics.map( property( 'meta.analytics' ) )
+		analytics: analytics.map( property( 'meta.analytics[0]' ) )
```

Comment:
There's already a test case, but it's correct either way.
I'm discussing it with @dmsnell in case this leads to regressions --well, future regressions, since I'm apparently the first one using `composeAnalytics`.